apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/model/Fiss.java

```diff
@@ -14,29 +14,49 @@
  * corresponding column enum class.
  */
 public class Fiss {
-
+  /** Admitting Dianosis Code is set to Inpatient Claim Admitting Diagnosis Code name. */
```

Comment:
Don't need to mention the initialized value, just what the field is for.

Reply:
done and cleaned up below them